src/plugins/janus_audiobridge.c

```diff
@@ -8777,9 +8779,61 @@ static void *janus_audiobridge_participant_thread(void *data) {
 					jitter_ticks = 0;
 				}
 				jitter_buffer_tick(participant->jitter);
+
 				if(ret != JITTER_BUFFER_OK) {
-					/* No packet in the jitter buffer? Move on the talking detection, if needed */
-					janus_audiobridge_participant_istalking(session, participant, NULL, NULL);
+					if(!first && participant->codec == JANUS_AUDIOCODEC_OPUS && lost_packets_gap <= JITTER_BUFFER_MAX_GAP_SIZE) {
```

Comment:
Please add a check here for `participant->muted`, otherwise when a participant is muted every buffer tick will generate a concealed packet.

Reply:
ok, added